Fix/buttons should not appear as links

getdave opened the pull request:
## Description
Closes https://github.com/WordPress/gutenberg/issues/15358#issuecomment-489615758.

Addresses issue from a11y report by removing link styling from buttons elements within the prepublish sidebar.

## How has this been tested?
* Open prepublish sidebar
* Verify the button text is not styled to visually appear as a link (eg: underlined and blue color)

## Screenshots <!-- if applicable -->

![Screen Shot 2019-05-06 at 15 42 55](https://user-images.githubusercontent.com/444434/57229129-a8cad500-7015-11e9-8af1-ae9851bb8187.png)

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. <!-- Check code: `npm run lint`, Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/ -->
- [ ] My code follows the accessibility standards. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/accessibility-coding-standards/ -->
- [ ] My code has proper inline documentation. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/ -->
- [ ] I've included developer documentation if appropriate. <!-- Handbook: https://wordpress.org/gutenberg/handbook/designers-developers/ -->

getdave committed:
Removes colors that visually imply a hyperlink

Addresses https://github.com/WordPress/gutenberg/issues/15358#issue-438944848

getdave committed:
Revert "Removes colors that visually imply a hyperlink"

This reverts commit 1aaec62d47cd95f3787e2f24d5e25b06bb6580a6.

getdave committed:
Removes link styling from buttons in prepublish sidebar

Addresses https://github.com/WordPress/gutenberg/issues/15358#issuecomment-489615758

jorgefilipecosta commented on packages/editor/src/components/post-publish-panel/style.scss:
```diff
@@ -54,10 +54,9 @@
 	font-weight: 400;
 	padding-left: 4px;
-	text-decoration: underline;
+	text-decoration: none;
```
Minor: I guess given that the element is a span, we don't need to add text-decoration: none; and we can just remove text-decoration: underline;.

kjellr replied:
Agreed, this PR works fine with the text-decoration rule deleted. 👍

jorgefilipecosta reviewed (approved):
Works as expected 👍

kjellr reviewed (approved):
This works for me! 

<img width="277" alt="Screen Shot 2019-05-07 at 9 20 15 AM" src="https://user-images.githubusercontent.com/1202812/57302599-926e5900-70a9-11e9-93a4-f9726cfd7511.png">

I agree with @jorgefilipecosta's note about the `text-decoration` rule, but once that's removed this should be good to merge in. 👍 

Thanks!

afercia commented:
Looks good from an a11y perspective, thanks for the PR! As already noted, `text-decoration: none;` seems redundant.

getdave committed:
Removes superfluous text decoration rule

Addresses https://github.com/WordPress/gutenberg/pull/15460#discussion_r281625728
